mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -192,8 +192,9 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7350 resetLastAttackedTicks ()V
 	METHOD method_7351 getBlockBreakingSpeed (Lnet/minecraft/class_2680;)F
 		ARG 1 block
-	METHOD method_7353 addChatMessage (Lnet/minecraft/class_2561;Z)V
+	METHOD method_7353 addGameMessage (Lnet/minecraft/class_2561;Z)V
 		ARG 1 message
+		ARG 2 infoMessage
```

Comment:
Isn't this like [`/title actionbar`](https://minecraft.gamepedia.com/Commands/title)? This could be `ARG 2 actionBar`.

Reply:
Action bar messages refer to info message type in the code, but "actionBar" is probably clearer?

Reply:
"Info message" doesn't give you the impression that it would be an action bar message.